lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -161,6 +161,12 @@ def main(argv=None):
                              'then the number of iterations may require '
                              'increasing, as there will be more coefficients '
                              'to solve for.')
+    parser.add_argument('--tolerance', metavar='TOLERANCE',
+                        type=np.float32, default=1,
```

Comment:
Is this a sensible default value?

Reply:
It's sensible based on the current use case, focusing on screen temperature where the CRPS value computed from the whole domain is large, so a tolerance of 1 is reasonable. The default tolerance of 1e-4 within scipy ([see optimize.py](https://github.com/scipy/scipy/blob/15e214e50ffd57125dd96a846e29657ae02758a2/scipy/optimize/optimize.py#L459)) is overly small for the current use case, so I've chosen a larger tolerance. Depending on the direction that the EMOS code is developed, we might prefer a different default, but this is reasonable for the current use case.

Reply:
OK.  I was confused by "tolerance" being used further down in the code with a value of 1e-4, which I think is the tolerance for the coefficient values themselves, whereas this is the tolerance on the CRPS.  Would it be sensible to specify at each level what the tolerance is on (in this docstring and further down) / how each "tolerance" is being used?

Reply:
Yes, I can add a bit more detail to clarify what the tolerance relates to. All the references to "tolerance" in this PR are with regards to the tolerance on the CRPS. As mentioned, I've used a tolerance of 1 as the default, but I have used a tolerance of 1e-4 for some of the testing. I can update the tests, if this is confusing.

Reply:
I think this doc-string needs to make clear that the tolerance is in units of, and relates to, typical values of the diagnostic, though that is perhaps obvious to someone who understands CRPS. Ergo, 1 is suitable for temperature because the values are typically ~300K, but it would be terrible for rainfall rates in ms-1.